### ISSUE TITLE
Support for Dynamics 365

### ISSUE THREAD

**Issue by ghost** [hybridgroup/cylon#360]
I saw that there is support for Salesforce already. Would it be possible to have support for Dynamics 365, PowerApps or Flow also ?

**Comment by deadprogram** [comment]
Hi @con-ssc all it would take is someone creating a new Cylon.js module with the Adaptor and Driver to to support it. We'd love to see those contributions!

**Comment by ghost** [comment]
Well, if I would be capable of doing it, you'd have a PR and not a Feature Request ;)

**Comment by deadprogram** [comment]
Indeed. :) Well, I am going to tag it as that. Perhaps some members of our community here have the same needs. We're not currently a user of any of those other CRM systems, at least as of yet, so not as easy to work on it.
